instrumentation/grizzly-2.3/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/grizzly/GrizzlyNetAttributesGetter.java

```diff
@@ -30,7 +32,21 @@ public String getTransport(HttpRequestPacket request) {
 
   @Nullable
   @Override
-  public String getProtocolName(HttpRequestPacket request) {
+  public String getNetworkTransport(
+      HttpRequestPacket request, @Nullable HttpResponsePacket response) {
+    Transport transport = request.getConnection().getTransport();
+    if (transport instanceof TCPNIOTransport) {
+      return "tcp";
+    } else if (transport instanceof UDPNIOTransport) {
```

Comment:
remove redundant else
```suggestion
    } 
    if (transport instanceof UDPNIOTransport) {
```